.github/workflows/manifests.yaml

```diff
@@ -0,0 +1,32 @@
+name: manifests-diff
+
+on:
+  pull_request:
+  push:
+    branches:
+    - main
+
+jobs:
+  manifests-diff:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout target branch of repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.base_ref }}
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: "go.mod"
+      - name: Generate rukpak.yaml installation manifest from target branch
+        run: make quickstart
+      - name: Rename manifest file for later comparison
+        run: mv rukpak.yaml rukpak-TODO.yaml
```

Comment:
WDYT about renaming these to help PR reviewers understand context a little better? For example `base-branch-manifest.yaml` and `pr-manifest.yaml`

Reply:
Agreed, I left the filename as `rukpak-TODO.yaml` and was going to have that be named after the branch.

Reply:
I've updated the branch so the manifest files will match the branches which generated them.